APMrover2/navigation.cpp

```diff
@@ -22,7 +22,7 @@ void Rover::navigate()
 	wp_distance = get_distance(current_loc, next_WP);
 
 	if (wp_distance < 0){
-		gcs_send_text(MAV_SEVERITY_CRITICAL,"<navigate> WP error - distance < 0");
+		gcs_send_text(MAV_SEVERITY_NOTICE,"<navigate> WP error - distance < 0");
```

Comment:
@gmorph The issue is how bad is bad :) From the enum of the severities NOTICE is An unusual event has occured, though not an error condition. This should be investigated for the root cause.

Is this enough or should it be raised to a higher level ? I don't think it's enough to: Action must be taken immediately. Indicates failure in a primary system.